### PR TITLE
v1.5.10 Update rewards response objects [BAC-2154]

### PR DESCRIPTION
**Pull request by jiajames**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dydxprotocol/v3-client",
-  "version": "1.5.9",
+  "version": "1.5.10",
```
Comment (samweinberg23) [inline package.json:3]: update pr title

File: src/types.ts [hunk 2]
```diff
@@ -504,6 +512,11 @@ export interface LiquidityRewards {
   estimatedRewards: string,
 }
 
+export interface StakedDYDXLiquidity {
```
Comment (samweinberg23) [inline src/types.ts:515]: feel like `StakedDYDX` could extend this interface which will require some interface naming changes

Comment (jiajames) [reply src/types.ts]: updated to `StakedDYDX` and `StakedDYDXIncludingFloor extends StakedDYDX` - don't think this is the greatest naming so if you have a suggestion i'll probably take it

File: package.json (hunk 1 above)
Comment (jiajames) [reply package.json:3]: done

**Review by samweinberg23 — APPROVED** [review]

**Pull request merged**